logger/logger.go

```diff
@@ -71,3 +44,84 @@ func GetLevel(levelStr string) (Level, error) {
 	}
 	return InfoLevel, fmt.Errorf("Unknown Level String: '%s', defaulting to NoLevel", levelStr)
 }
+
+// basic Logger is the default global
+var globalLogger Logger = NewLogger()
+
+func SetGlobalLogger(logger Logger) {
+	globalLogger = logger
+}
+
+func SetGlobalLevel(lvl Level) {
+	globalLogger.SetLevel(lvl)
+}
+
+func Info(args ...interface{}) {
+	globalLogger.Log(InfoLevel, "", args, nil)
+}
+func Infof(template string, args ...interface{}) {
```

Comment:
Is template the right word? Should this nor be format?

Reply:
what would you propose? `format` ?